### ISSUE TITLE
Launch Version 2

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#500]
## Problem
Version 2 isn't launched!

## Overview
Version 2 represents our full-featured product built to compete with significant accessibility platforms. Our goal is to become a mission-driven company focused, first and foremost, on helping people with disabilities thrive while delivering the technically best web accessibility testing and remediation features.

## Process
Feature Development has the following process (we'll tick off items as we work through them):

- [x] User Story Development(#501): Develop descriptions of the users we're building for. These should be educated by past work. Note: All users work toward the same goal: fixing accessibility issues.
- [x] Design Phase (#492): We'll mockup features in a visual and text-based design document that helps work out a cohesive user experience for sighted users.
- [x] Mock Development: We'll develop mocks for user testing. #518 and #517 #518
- [ ] Production Development: We'll develop all features #520 
- [ ] Final testing with screen reader users.
- [ ] @bbertucc closes this issue.

**Comment by bbertucc** [comment]
We've set the roadmap for V2 release: https://docs.google.com/document/d/1YNFq4A5cLDCxjYbJs00NoCb6_VvG1N4FcJB66sKcg70/edit?usp=sharing
